libtcmu.c

```diff
@@ -835,6 +835,11 @@ void tcmu_dev_set_private(struct tcmu_device *dev, void *private)
 	dev->hm_private = private;
 }
 
+const char *tcmu_dev_get_td_name(struct tcmu_device *dev)
```

Comment:
Is "td" short of tcmu_dev? Could you rename to tcmu_dev_get_lio_name or tcmu_dev_get_kern_name to reflect is the lio/kernel backend name.